split.js

```diff
@@ -38,9 +38,11 @@ async function processAction(msg, conf) {
     }
 
     debug('%s parts to emit found', result.length);
-    result.forEach(async elem => {
-        await this.emit('data', messages.newMessageWithBody(elem));
-    });
+    for (let i = 0; i < result.length; i++) {
```

Comment:
Cleanest syntax would be
```
for (let value of result){
  await this.emit('data', messages.newMessageWithBody(value));
}
```
I don't think you need to add the null check.

Reply:
Done